CMakeLists.txt

```diff
@@ -0,0 +1,167 @@
+# largely lifted/referenced from
+# https://github.com/trailofbits/remill/blob/master/CMakeLists.txt
+project(grr)
+cmake_minimum_required (VERSION 3.2)
+
+enable_language(C)
+enable_language(CXX)
+enable_language(ASM)
+
+# What compilers are we using?
+set(CMAKE_C_COMPILER "clang-3.8")
```

Comment:
This I think is the one change I'd like to see. Can you see if you can detect if the cmake compiler is not a variant of clang, and error out? Hard-coding clang 3.8 should not be necessary anymore.

Reply:
Done in 395631b.